templates/vstore_cart_template.php

```diff
@@ -9,11 +9,11 @@
 <table class="table table-hover table-striped cart">
 <thead>
 	<tr>
-		<th>Product</th>
+		<th>'.LAN_VSTORE_CART_005.'</th>
 		<th> </th>
-		<th>Quantity</th>
-		<th class="text-right text-end">Price</th>
-		<th class="text-right text-end">Total</th>
+		<th>'.LAN_VSTORE_CART_010.'</th>
+		<th class="text-right text-end">'.LAN_PLUGIN_VSTORE_PRICE.'</th>
+		<th class="text-right text-end">'.LAN_VSTORE_GEN_013.'</th>
```

Comment:
```
<th>'.LAN_VSTORE_CART_010.'</th>
<th class="text-right text-end">'.LAN_PLUGIN_VSTORE_PRICE.'</th>
<th class="text-right text-end">'.LAN_VSTORE_GEN_013.'</th>
```

This is a good example of why we should simply use the LAN_VSTORE_XXX format for all frontend terms.